### ISSUE TITLE
How to add the go-ssb-room plugin to go-ssb-pub

### ISSUE THREAD

**Issue by Scott20222** [cryptoscope/ssb#139]
hello,cryptix, We tried to use manyverse to communicate with go-ssb, but failed, but we can communicate through go-ssb-room, you are involved in both projects, is there any plan to integrate room into pub, I see room FAQ There is a nodejs plugin(https://github.com/staltz/ssb-room/blob/master/DEVELOPERS.md) that can be integrated into pub, but there is no description of the go version, thanks!

**Comment by staltz** [comment]
Room and pubs are two distinct types of servers, I don't see why we would add one into the other.

**Comment by cryptix** [comment]
Agreed with staltz. Technically feasable to embedd the go-ssb code into room but not necessary at all.

If you want a pub, just host a pub.

If you want a go-ssb peer inside your room you can just do this:

```bash
sbotcli call conn.replicate '@uMYDVPuEKftL4SzpRGVyQxLdyPkOiX7njit7+qT/7IQ=.ed25519'
sbotcli call conn.connect 'net:hermies.club:8008~shs:uMYDVPuEKftL4SzpRGVyQxLdyPkOiX7njit7+qT/7IQ='
```

the `conn.replicate` line is a bit of a misnomer (since there is no feed to replicate) but it allows the go-ssb node to connect to the room server, since it only allows connect from known peers. Afterwards it will be available for other peers in the room to dial it (if it knows them).

**Comment by Scott20222** [comment]
Yes, the main reason is that using manyverse to communicate with go-ssb is not normal, but it can communicate quickly with go-ssb-room. Manyverse has a built-in room client, which can quickly establish a connection with the room server on the cloud. Manyverse crashes when manyverse establishes communication with go ssb pub. I wonder if it is because there is no room server in pub.
